packages/blob-reader/helper.js

```diff
@@ -74,34 +74,19 @@ function getPath(el) {
   }
 
   // When current page is a gist, get path from blob name
-  if (!ret && isGist()) {
+  if (isGist()) {
     ret = $('.gist-blob-name', el.parentElement).text().trim();
     if (ret && !ret.startsWith('/')) {
       ret = `/${ret}`;
     }
   }
 
   // when page has pull request comment(s)
-  const $fileHeader = $('.file-header', el.parentElement);
+  const $fileHeader = $('summary', el.parentElement.parentElement);
   if (!ret && $fileHeader.length) {
-    let repoPath = '';
-    let filePath = '';
-
-    if ($('a.file-action', $fileHeader).length) {
-      // comment is outdated
-
-      filePath = $('.file-info', $fileHeader).text();
-      repoPath = $('a.file-action', $fileHeader).attr('href');
-    } else {
-      // comment is up-to-date
-
-      filePath = $('a', $fileHeader).text();
-      repoPath = $('a', $fileHeader).attr('href');
-    }
-
-    if (repoPath && filePath) {
-      ret = mergeRepoAndFilePath(repoPath, filePath);
-    }
+    const filePath = $('a', $fileHeader).text();
+    const repoPath = $('a', $fileHeader).attr('href');
+    ret = mergeRepoAndFilePath(repoPath, filePath);
```

Comment:
This was another change to the dom that caused code snippets part of a review to be broken because OctoLinker was unable to construct the file path which is being used to invoke the related plugin for this file extension.